feat: add documentation for string keys (#4938)
camunda/camunda-docs: docs/self-managed/operational-guides/update-guide/860-to-870.md

- `ASSIGN`
- `ASSIGNING`
- `ASSIGNED`

## REST API key attributes change to `string`

> With 8.7 the default REST API key attribute type changes from `number (int64)` to default `string`.

Reviewer: ```suggestion
With 8.7, the default REST API key attribute type changes from `number (int64)` to `string`.
```

> This means key properties, that is properties with `key` suffix, for instance `processDefinitionKey`, `tenantKey` etc.
> (only exception is `correlationKey`) are now defined with type `string` for the default `application/json` content type.

Reviewer: ```suggestion
Key properties have a `key` suffix, serving as the technical unique identifier for entities, such as `processDefinitionKey` and `tenantKey`, with `correlationKey` being the only exception. Their type now defaults to `string` for the default `application/json` content type.
```

Author: Sounds good, I'll just add a minor change.

Our API can still work with key attributes as numbers using different content type headers.  
See the [overview about API Key Attributes][camunda8-api-overview] for more details.

:::warning Still using number keys?
If you still wish to use keys as `number (int64)` types, you have to include the
`application/vnd.camunda.api.keys.number+json` header in your client requests.
:::

> If you are using an older, 8.6 or 8.5 Java Client, then C8 API will recognize this, and automatically return key
> attributes as `integer (int64)`, meaning you don't need to add the extra content type header.

Reviewer: ```suggestion
If you are using an 8.6 or 8.5 Java Client, the C8 API will recognize this and automatically return key
attributes as `integer (int64)`. You don't need to add the extra content-type header to your client requests.
```
